CHANGELOG.md

```diff
@@ -1,6 +1,9 @@
-0.17.0 / TBC
+0.16.2 / 2017-10-3
 ===================
-- Update all available npm dependencies
+- Update ADR guidance/superseded mongo ADR
+- Update all stable npm dependencies
```

Comment:
The inclusion of `stable` makes me wonder what the unstable dependencies were and why they weren't updated.

Reply:
There was a discussion about updating node to `8.6.0` and why that wasn't done, but I guess that is distinct from dependencies?

Reply:
Yeah, it isn't an npm dependency and the result was no change to the code base/application so there'd be no need to mention it in the release notes.